python/ray/tests/test_draining.py

```diff
@@ -127,10 +127,10 @@ def test_scheduling_placement_groups_during_draining(ray_start_cluster):
     """Test that the draining node is unschedulable for new pgs."""
     cluster = ray_start_cluster
     cluster.add_node(num_cpus=1, resources={"node1": 1})
+    ray.init(address=cluster.address)
```

Comment:
This makes sure node1 is the head node. Otherwise, the head node can be one of the three randomly.